tests/python/relax/test_codegen_cudnn.py

```diff
@@ -36,12 +36,8 @@ def reset_seed():
 
 has_cudnn = tvm.get_global_func("relax.ext.cudnn", True)
```

Comment:
Do we still need `has_cudnn` now that it isn't used in `pytest.mark.skipif`?

Reply:
You re absolutely right. Will remove it.